### PR TITLE
feat: Community Edition removal devicemanager.list 

### PR DESCRIPTION
**Pull request by Zeno-sole**
移除devicemanager.list,该软件源与打印管理器源重复

Log:

### CODE REVIEW

File: debian/deepin-devicemanager.postinst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+export APTFILEPATH="/etc/apt/sources.list.d/devicemanager.list"
+OS_VERSION=/etc/os-version
+if [[ ! -e $OS_VERSION ]]
+then
+        if [[ ! -e $DEEPIN_VERSION ]]
```
Comment (UTsweetyfish) [inline debian/deepin-devicemanager.postinst:6]: DEEPIN_VERSION?

**Review by UTsweetyfish — CHANGES_REQUESTED** [review]

**Review by wyu71 — APPROVED** [review]
/+1

**Pull request merged**